install-xamarin-android-ios.sh

```diff
@@ -1,5 +1,6 @@
 XAMARIN_IOS_VERSION=11.10.1.178
 XAMARIN_ANDROID_VERSION=8.3.0-19
+XAMARIN_MAC_VERSION=4.8.0.22
```

Comment:
I think you will have to take the lower version [4.5.0.280](https://docs.microsoft.com/en-us/xamarin/mac/release-notes/4/4.5) all above require XCode 9.4 or higher.

Reply:
Ok. I changed it.